### PR TITLE
[risk=low][RW-7863] More UI tweaks for AdminUserProfile

### PR DESCRIPTION
**Pull request by jmthibault79**
Description:

---
**PR checklist**

- [ ] This PR meets the Acceptance Criteria in the JIRA story
- [ ] The JIRA story has been moved to Dev Review
- [ ] This PR includes appropriate unit tests
- [ ] I have run and tested this change locally
- [ ] I have run the E2E tests on ths change against my local UI and/or API server with `yarn test-local` or [yarn test-local-devup](https://github.com/all-of-us/workbench/blob/main/e2e/README.md#examples)
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
- [ ] If this includes an API change, I have updated the appropriate Swagger definitions and notified API consumers
- [ ] If this includes a new feature flag, I have created and linked new JIRA tickets to (a) turn on the feature flag and (b) remove it later

### CODE REVIEW

File: ui/src/app/pages/admin/admin-user-common.tsx [hunk 1]
```diff
@@ -149,7 +151,7 @@ export const getInitialCreditLimitOptions = (
   );
 };
 
-export const getInitalCreditsUsage = (profile: Profile): string => {
+export const getInitialCreditsUsage = (profile: Profile): string => {
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/admin-user-common.tsx:154]: spelling

File: ui/src/app/pages/admin/admin-user-profile.tsx [hunk 2]
```diff
@@ -91,7 +91,7 @@ const styles = reactStyles({
   },
   uneditableFields: {
     height: '221px',
-    width: '601px',
+    width: '650px',
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/admin-user-profile.tsx:94]: 700 looks decent too

File: ui/src/app/pages/admin/admin-user-profile.tsx [hunk 3]
```diff
@@ -107,7 +107,7 @@ const UneditableField = (props: {
   label: string;
   value: string | JSX.Element;
 }) => (
-  <FlexColumn>
+  <FlexColumn style={{ paddingTop: '1em', width: '300px' }}>
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/admin-user-profile.tsx:110]: better spacing

File: ui/src/app/pages/admin/admin-user-profile.tsx [hunk 4]
```diff
@@ -124,22 +124,25 @@ const UneditableFields = (props: { profile: Profile }) => {
     ) : (
       accessTierShortNames.map(displayNameForTier).join(', ')
     );
+
   return (
-    <FlexRow style={styles.uneditableFields}>
```
Comment (jmthibault79) [inline ui/src/app/pages/admin/admin-user-profile.tsx:128]: overflow caused some problems with this arrangement

**Review by NehaBroad — APPROVED** [review]
Tested

**Review by aweng98 — APPROVED** [review]

**Pull request merged**